crates/katana/src/main.rs

```diff
@@ -102,3 +108,20 @@ ACCOUNTS SEED
 
     println!("\n{address}\n\n");
 }
+
+pub async fn shutdown_handler(sequencer: Arc<impl Sequencer>, config: KatanaArgs) {
+    println!(); // Extra line break for ^C keypress message
+    println!("Katana shutting down.");
+
+    if let Some(path) = config.dump_state {
+        let state = (*sequencer).backend().dump_state().await.unwrap();
+        match fs::write(path.clone(), state) {
+            Ok(_) => {
+                println!("State dumped to file {:?}", path)
```

Comment:
use the logging macro

```suggestion
                info!("Successfully dumped state on shutdown")
```

Reply:
also i believe we dont have to display the path as the user would already be passing it in the cli anyway